addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
@job13er - You are welcome. I think the imports/destructing are separate enough that if we wanted to break them out it would be fine. However, I do see your point about the trailing commas. I am torn on this one. I see tradeoffs with each approach. In this case, I recommend that we just pick one and move forward.